poll/public/handlebars/poll_studio.handlebars

```diff
@@ -0,0 +1,22 @@
+<script id="poll-form-component" type="text/html">
+{{#each items}}
+<li class="field comp-setting-entry is-set poll-{{noun}}-studio-item">
+    <div class="wrapper-comp-setting">
+        <label class="label setting-label poll-setting-label" for="{{noun}}-label-{{key}}">{{noun}}</label>
+        <input class="input setting-input" name="{{noun}}-label-{{key}}" id="{{noun}}-label-{{key}}" value="{{text}}" type="text" /><br />
+        {{#if image}}
+        <label class="label setting-label" for="{{noun}}-img-{{key}}">Image URL</label>
+        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" />
+        {{/if}}
+        <div class="poll-move">
+            <div class="poll-move-up">&#9650;</div>
+            <div class="poll-move-down">&#9660;</div>
+        </div>
+    </div>
+    <span class="tip setting-help">
+        {{#if image}}This must have an image URL or text, and can have both.{{/if}}
```

Comment:
I added a form field and a comment to strongly encourage filling it in, but didn't go all the way to make it mandatory to support the legacy content that doesn't have any alt texts applied.  We can't verify whether people entered anything useful anyway.